package.json

```diff
@@ -54,7 +54,7 @@
     "@types/crypto-js": "^3.1.43",
     "crypto-js": "^3.1.9-1",
     "js-joda": "^1.6.2",
-    "js-xpx-catapult-library": "^0.1.0",
+    "js-xpx-catapult-library": "^0.2.0",
```

Comment:
js-xpx-chain-library

Reply:
there is many more places where this has to be changed - it will be done as separate commit following this one